src/client/services/Network.ts

```diff
@@ -3,10 +3,16 @@ import Phaser from 'phaser'
 import { IOfficeState, IPlayer } from '../../types/IOfficeState'
 import { Message } from '../../types/Messages'
 
+enum Event {
+  PLAYERJOINED = 'player-joined',
```

Comment:
GG sorry I was not clear, for these constants/enums, it should look like: `PLAYER_JOINED`

Reply:
Done!